chrome/content/zotero/xpcom/style.js

```diff
@@ -399,6 +431,72 @@ Zotero.Styles = new function() {
 			}
 		});
 	}
+	
+	/**
+	 * Populate menulist with locales
+	 */
+	this.populateLocaleList = function(menulist, prefLocale) {
+		if(!_initialized) this.init();
+		
+		var doc = menulist.ownerDocument;
+		var popup = doc.createElement('menupopup');
+		menulist.appendChild(popup);
```

Comment:
Open preferences -> Reset Styles. This ends up calling `populateQuickCopyList`, which, in turn, ends up calling this function outside of init call. And if you look at [`buildQuickCopyFormatDropDown`](https://github.com/zotero/zotero/blob/15108eea3f099a5a39a145ed55043d1ed9889e6a/chrome/content/zotero/preferences/preferences_export.js#L61), it makes sure to clear the list before populating it.